tools/accuracy_checker/requirements-extra.in

```
@@ -15,8 +15,7 @@ imagecodecs~=2021.11.20;python_version=="3.7"
 imagecodecs~=2022.2.22;python_version>="3.8"
 
 # reid
-# python3.6 dropped in v1.0
-scikit-learn~=0.24.1
+scikit-learn~=1.2.0
```

Can we remove upper bound
```suggestion
scikit-learn>=1.2.0
```
?

---

I'm okay with it, as long as there's a `=` :)